unicodetools/data/security/dev/data/source/removals.txt

```diff
@@ -846,3 +846,21 @@ AB68; Technical
 # UTC-179-C39: Change the Identifier_Type of A9CF to Limited_Use Uncommon_Use, removing Exclusion.
 # (A code fix removed Exclusion from Limited_Use characters, but adding Uncommon_Use needs an override.)
 A9CF; Limited_Use Uncommon_Use
+
+# Unicode 17
+09FF; Technical # L2/24-013
+0B53..0B54; Uncommon_Use # L2/24-068
+0C5C; Obsolete # L2/22-006
+0CDC; Obsolete # L2/22-006
+1ACF..1ADD; Technical # L2/24-105, L2/23-188, L2/23-208, L2/23-206, L2/23-238
+1AE0..1AEA; Technical # L2/24-080
+A7CE..A7CF; Uncommon_Use # L2/23-219
+A7D2; Obsolete # L2/23-135
+A7D4; Obsolete # L2/23-135
+10EC5; Technical # L2/23-019
+10EC6; Technical # L2/23-121
+10EC7; Uncommon_Use # L2/24-055
+10EFB; Technical # L2/22-281R
+16FF2..16FF6; Technical # L2/23-284
+2B73A..2B73E; Uncommon_Use # L2/25-031
+\p{block=CJK_Unified_Ideographs_Extension_J}; Uncommon_Use # L2/25-031
```

Comment:
@roozbehp I've added L2 document numbers to removals.txt. Please review and advise if any changes are needed.

Reply:
Updated in https://github.com/unicode-org/unicodetools/pull/1062/commits/1b2d045dab3dca6aad8c394ebc35d945ebd2b5ea

Reply:
We still have the "work in progress" label, but it looks like this might be done?
I could rubber stamp it.

Reply:
> We still have the "work in progress" label, but it looks like this might be done? I could rubber stamp it.

@markusicu I have not reviewed the Python changes and I don't plan to. So please take a look at those.

Reply:
> > We still have the "work in progress" label, but it looks like this might be done? I could rubber stamp it.
> 
> @markusicu I have not reviewed the Python changes and I don't plan to. So please take a look at those.

I at least glanced at every file. Once you are happy with the ID_Type changes and documentation, we are all good.